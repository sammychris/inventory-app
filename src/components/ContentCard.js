import React from 'react';
import styled from 'styled-components';


const ContentCard = ({children}) => {
    return (
        <Container>
            {children}
        </Container>
    );
  }


export default ContentCard;

const Container = styled.div`
    min-height: 200px;
    width: 100%;
    border-radius: 10px;
    padding: 12px 15px 12px 15px;
    background: #fff;
    color: #000;
    margin-top: 20px;
`

const Title = styled.span`
    font-family: Poppins;
    font-size: 15px;
    font-weight: 400;
    letter-spacing: 0em;
`;

const ContentValue = styled.div`
    font-family: Poppins;
    font-size: 40px;
    font-weight: 400;
    line-height: 60px;
    letter-spacing: 0em;
    text-align: center;
`;