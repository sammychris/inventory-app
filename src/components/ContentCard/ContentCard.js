import React from 'react';
import styled from 'styled-components';


const ContentCard = ({children}) => {
    return (
        <Container>
            {children}
        </Container>
    );
  }


export default ContentCard;

const Container = styled.div`
    min-height: 200px;
    width: 100%;
    border-radius: 10px;
    padding: 12px 15px 12px 15px;
    background: #fff;
    color: #000;
    margin-top: 20px;
`