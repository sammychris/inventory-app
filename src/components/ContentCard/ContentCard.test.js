import { shallow } from "enzyme";
import ContentCard from "./ContentCard";

const wrapper = shallow(<ContentCard />);

test("expect to render a ContentCard", () => {
  expect(wrapper.length).toEqual(1);
});

test("expect to make a snap shot of ContentCard component", () => {
  expect(wrapper).toMatchSnapshot();
});