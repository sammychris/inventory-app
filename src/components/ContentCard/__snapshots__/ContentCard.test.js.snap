// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to make a snap shot of ContentCard component 1`] = `ShallowWrapper {}`;
